docs/mobile-apps/features/biometric-authentication.md

```diff
@@ -17,6 +17,7 @@ Sauce Labs provides biometrics interception for our customers to:
 - Verify that an app responds as expected to biometrics settings (e.g., enabled/disabled and successful/unsuccessful).
 
 :::caution
+Please make sure you have a debuggable AND non-obfuscated version of your application uploaded to Mobile App Storage. 
```

Comment:
Can we provide some guidance for our customers on how to do that? Just in a few steps?

Reply:
yup, lets create a new page under mobile app storage and merge it with this:
https://docs.saucelabs.com/mobile-apps/automated-testing/ipa-files/ 

not in this PR though. when the two pages are merged we will add a link to the new
https://docs.saucelabs.com/mobile-apps/ipa-apk/files